windows/ReactTestApp/MainPage.cpp

```diff
@@ -87,14 +92,12 @@ namespace winrt::ReactTestApp::implementation
     MenuFlyoutItem MainPage::MakeComponentMenuButton(::ReactTestApp::Component &component)
```

Comment:
Probably should've noticed this earlier but can `component` be `const`?
```suggestion
    MenuFlyoutItem MainPage::MakeComponentMenuButton(::ReactTestApp::Component const &component)
```